msa/tb-node/docker/start-tb-node.sh

```diff
@@ -15,14 +15,21 @@
 # limitations under the License.
 #
 
-CONF_FOLDER="/config"
 jarfile=${pkg.installFolder}/bin/${pkg.name}.jar
 configfile=${pkg.name}.conf
 run_user=${pkg.user}
 
-source "${CONF_FOLDER}/${configfile}"
+CONF_FOLDER="/config"
+if [ -d "${CONF_FOLDER}" ]; then
+  LOGGING_CONFIG="${CONF_FOLDER}/logback.xml"
+  source "${CONF_FOLDER}/${configfile}"
+  export LOADER_PATH=${CONF_FOLDER},${LOADER_PATH}
+else
+  CONF_FOLDER="/usr/share/${pkg.name}/conf"
+  LOGGING_CONFIG="/usr/share/${pkg.name}/conf/logback.xml"
+  source "${CONF_FOLDER}/${configfile}"
```

Comment:
Is it ok we don't add conf folder to LOADER_PATH?

Reply:
It is already added. See https://github.com/thingsboard/thingsboard/blob/master/application/src/main/conf/thingsboard.conf#L23